src/micro_endpoint.c

```diff
@@ -334,7 +279,7 @@ micro_clone_endpoint_config(microEndpointConfig **out, microEndpointConfig *cfg)
     microEndpointConfig *new_cfg = NULL;
 
     if (out == NULL)
-        return micro_ErrorInvalidArg;
+        return microError_Wrapf(err, "failed to clone endpoint config: '%s'", cfg->Name);
```

Comment:
I checked the function, and it will be a no-op if `err` is `NULL`, which it is here. So you should use `micro_ErrorInvalidArg` here instead of `err` (which was originally the case, but I guess it is a copy/paste to get the config name included).